reactor-netty-core/src/main/java/reactor/netty/resources/PooledConnectionProvider.java

```diff
@@ -220,6 +233,34 @@ protected static void logPoolState(Channel channel, InstrumentedPool<? extends C
 				t == null ? "" : t);
 	}
 
+	final void scheduleInactivePoolsDisposal() {
+		if (!disposeInterval.isZero()) {
+			Schedulers.parallel()
+			          .schedule(this::disposeInactivePoolsInBackground, disposeInterval.toNanos(), TimeUnit.NANOSECONDS);
```

Comment:
I wonder about the ns resolution for the interval. I'm pretty sure no JDK Executor can handle scheduling that precise, but I guess for tests with VTS it could be useful...

But milliseconds would be good enough, wouldn't they? Plus since the inactivity info is in seconds, it doesn't make much sense to check at nanosecond intervals, does it? (we can lag behind a few milliseconds, if the pool has been inactive for eg. 5 seconds it won't make a huge difference).

Reply:
ok I'll change it